daml-script/test/JsonTestRunner.hs

```diff
@@ -0,0 +1,54 @@
+-- Copyright (c) 2020 The DAML Authors. All rights reserved.
```

Comment:
Please rebase for the COPY change (and you'll need to update this).

Reply:
Thanks! Rebased and updated.